lib/jekyll/commands/doctor.rb

```diff
@@ -44,6 +45,24 @@ def healthy?(site)
           ].all?
         end
 
+        def outdated_dependency_check(site)
+          return true unless can_test_dependency_health?(site)
+
+          Dir.chdir(site.source) do
+            exe = Gem.bin_path("bundler", "bundle")
+            output = Jekyll::Utils::Exec.run("ruby", exe, "outdated", "--parseable")[-1]
```

Comment:
This [-1] is confusing. Can we just do what you did before with the splat and the underscore prefixed variable?

You should be checking whether bundler successfully exited anyway, I would presume.